bigquery_etl/cli/query.py

```diff
@@ -163,6 +171,10 @@ def create(name, sql_dir, project_id, owner, init):
         description="Please provide a description for the query",
         owners=[owner],
         labels={"incremental": True},
+        bigquery=BigQueryMetadata(
+            time_partitioning=PartitionMetadata(field="", type=PartitionType.DAY),
+            clustering=ClusteringMetadata(fields=[]),
+        ),
```

Comment:
For explicitly adding time_partitioning and clustering configuration fields to newly created `metadata.yaml` files